spec/implementations/pto/fetch_pto_from_drive_spec.rb

```
@@ -0,0 +1,63 @@
+    ])
+
+    allow(Google::Apis::SheetsV4::SheetsService).to receive(:new).and_return(fake_service)
+    allow(fake_service).to receive(:authorization=)
+    allow(fake_service).to receive(:get_spreadsheet_values).and_return(fake_response)
+
+    allow(mocked_shared_storage_reader).to receive(:read).and_return(
+        instance_double(Bas::SharedStorage::Types::Read, id: 1, data: { key: 'value' }, inserted_at: Time.now)
+    )
+
+    allow(mocked_shared_storage_writer).to receive(:write).and_return(
+        [{ 'status' => 'success', 'id' => 1 }]
```

RuboCop (1.75.5)

[convention] Indent the first argument one step more than the start of the previous line.

(Layout/FirstArgumentIndentation)